styles/styles.css

```
@@ -51,56 +53,398 @@ h1, h2, h3, h4, h5, h6 {
 
 /* --------------- 4.1 Header ------------------------- */
 
+.outer-content-container-header {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-content-container-header {
+    flex-basis: 600px;
+    margin-inline: 20px;
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    flex-shrink: 1;
+    flex-grow: 0;
+}
+
 header {
+    background-color: #4845E4;
+    /*border-style: solid;*/
+    /*border-color: red;*/
+    position: relative;
     display: flex;
     flex-direction: column;
-    background: #4845E4;
-    color: white;
-    height: 400px;
-    justify-content: center;
-    border-style: solid;
-    border-color: red;
 }
 
-nav ul {
+.nav-bar {
```

```suggestion
.nav-bar ul {
```
Anders staan alsnog de listitems onder elkaar